manifests/r/Radionomy/Winamp/5.9/Radionomy.Winamp.installer.yaml

```diff
@@ -5,6 +5,8 @@ PackageIdentifier: Radionomy.Winamp
 PackageVersion: 5.9
 MinimumOSVersion: 10.0.0.0
 Scope: machine
+InstallModes:
+- interactive
```

Comment:
Packages must include a silent mode in order to be added

Reply:
I see, can't figure out how to install this in silent mode though 🤔

Reply:
It won't install silently because it has a dependency on `Microsoft.VC++2015-2019Redist-x86` version `14.29.30139.0`. 

@denelon can tag this with the Blocking-Issue